docs/reverse-proxy.md

```diff
@@ -0,0 +1,144 @@
+# Bento behind a proxy
+
+A Bento instance is typically deployed on a dedicated VM with a public IP address.
+
+By creating DNS `A` or `CNAME` records pointing hostnames to the VM's IP, deployers
+can obtain SSL certificates from Let's Encrypt using certbot.
+
+Having a Bento instance on a VM with a public IP is convenient in terms of deployment,
+but is a security trade-off, as it exposes the VM's IP and ports to attackers.
+
+A more secure deployment can be achieved by using a VM with no public IP to host Bento
+and proxy the trafic through a reverse-proxy.
+
+## Using a reverse proxy in front of Bento
+
+### Deployment overview
+As an example, let's assume we have the following hostnames to deploy:
+- Public: `sandbox.bento.example.com`
```

Comment:
i would use "Public dashboard" or something, otherwise it's a bit confusing vs. talking about public IPs/private IPs